sbi/inference/snpe/snpe_base.py

```diff
@@ -157,16 +157,21 @@ def __call__(
 
         for round_, num_sims in enumerate(num_sims_per_round):
```

Comment:
Not 100% sure, but shouldn't we start counting from `round_=1` now?

Reply:
The convention in the code right now is that we start from round 0. Should we change it to round 1?

Reply:
@meteore @janfb

Reply:
I would leave it with initial round idx zero. is there a specific reason for changing to start counting with one?